client/asset/dcr/dcr.go

```diff
@@ -1658,18 +1658,52 @@ func (dcr *ExchangeWallet) sendCoins(addr dcrutil.Address, coins asset.Coins, va
 	return tx, uint64(txOut.Value), err
 }
 
-// signTx attempts to sign all transaction inputs.
+// msgTxFromHex creates a wire.MsgTx by deserializing the hex transaction.
+func msgTxFromHex(txHex string) (*wire.MsgTx, error) {
+	msgTx := wire.NewMsgTx()
+	if err := msgTx.Deserialize(hex.NewDecoder(strings.NewReader(txHex))); err != nil {
+		return nil, err
+	}
+	return msgTx, nil
+}
+
+func msgTxToHex(msgTx *wire.MsgTx) string {
+	buf := bytes.NewBuffer(make([]byte, 0, msgTx.SerializeSize()))
+	if err := msgTx.Serialize(buf); err != nil {
+		return ""
+	}
```

Comment:
Not super important, but *wire.MsgTx has a Bytes() method that does this
https://github.com/decred/dcrd/blob/f1d6b2fc10bad0a4c76bcd43932fe3236db9d538/wire/msgtx.go#L975-L983

Reply:
Good call, updated.